mobile/examples/super_resolution/ios/ORTSuperResolution/ORTSuperResolution/ContentView.swift

```diff
@@ -38,7 +38,9 @@ struct ContentView: View {
                         Text("Output high resolution image: ").frame(width: 350, height: 40, alignment:.leading)
```

Comment:
nit: would 'higher resolution image' lower user expectations of extra special magic happening?